index.js

```diff
@@ -1,14 +1,29 @@
 function calculate(numberOne, operation, numberTwo) {
+  if (numberOne === undefined) {
```

Comment:
Great work. Check out the link to MDN for undefined that's in the test file. Or otherwise search "mdn undefined" which will take you to the same page.
There is a small trick mentioned there about checking for undefined.